test/functional/user_mailer_test.rb

```diff
@@ -329,22 +311,13 @@ def test_reply_posted_message_id
   end
 
   context('#issue_add') do
-    should 'send one email per recipient' do
```

Comment:
This test was completely moved over to RSpec.

Note that the name of this `should` is a lie. It does test that a mail is sent. It **does not** test anything related to the number of recipients and how that correlates to the number of mails, even though the name implies it.

When you look very closely, it is obvious why: You can only specify a single recipient to the method in question. So you will have to call it multiple times for multiple recipients (which the implementation does).